settings/example_track_settings.yml

```diff
@@ -0,0 +1,86 @@
+# Tracking mode: either "2D" (uses projected Z-slices) or "3D" (full volumetric).
+mode: "2D"
+
+# Z-slice range to extract. Use [0, 0] to auto-compute a centered slice.
+z_slices: [0, 0]
+
+# Name of the target channel used to name the tracking output label channel.
+target_channel: nuclei_prediction
+
+# Input image configurations.
+# Each entry defines one dataset with its path and channels to process.
+input_images:
+
+  # First input: predicted nuclei and membrane channels.
+  - path: /path/to/virtual_staining.zarr/*/*/*
```

Comment:
This field is redundant with the `input_position_dirpaths` argument of the tracking CLI. Paths to images should be provided only ones, during the CLI call